code/datums/components/spirit_holding.dm

```diff
@@ -8,6 +8,8 @@
 	var/attempting_awakening = FALSE
 	///mob contained in the item.
 	var/mob/living/simple_animal/shade/bound_spirit
+	///a thing can only get this component once
+	dupe_mode = COMPONENT_DUPE_UNIQUE
```

Comment:
vars that already exist on an object should be before new vars

Reply:
ty ty